packages/scripts/src/submission_export.js

```diff
@@ -33,7 +33,7 @@ export const formExportColumnHeaders = [
 
   // Preferences
   { id: 'deployAnywhere', title: 'Deploy Anywhere' },
-  { id: 'VancouverRegion', title: 'Vancouver Region' },
+  { id: 'VancouverCostal', title: 'Vancouver/Coastal' },
```

Comment:
Something to think about is how we'll handle changing copy in the future without changing the IDs, no biggie right now because we're pre-live